Tests/MeshLib/TestFindElementsInRadius.cpp

```diff
@@ -0,0 +1,247 @@
+/**
+ * @copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/LICENSE.txt
+ */
+
+#include <gtest/gtest.h>
+#include <autocheck/autocheck.hpp>
+#include <memory>
+#include <numeric>
+
+#include "BaseLib/makeVectorUnique.h"
+#include "BaseLib/CPUTime.h"
+#include "MathLib/GeometricBasics.h"
+#include "MeshLib/Elements/Element.h"
+#include "MeshLib/Mesh.h"
+#include "MeshLib/MeshGenerators/MeshGenerator.h"
+#include "MeshLib/Node.h"
+
+#include "Tests/AutoCheckTools.h"
+
+using namespace MeshLib;
+namespace ac = autocheck;
+
+struct MeshLibFindElementInRadius : public ::testing::Test
+{
+
+    ac::gtest_reporter gtest_reporter;
+};
+
+TEST_F(MeshLibFindElementInRadius, ZeroRadius)
```

Comment:
I'd always appreciate some description of the unit tests. E.g., `// for zero radius only the start element itself shall be returned.`